cg/cli/set/base.py

```diff
@@ -266,8 +266,12 @@ def sample(
         if not (yes or click.confirm(CONFIRM)):
             continue
 
-        setattr(sample_obj, new_key, new_value)
-        _update_comment(_generate_comment(new_key, old_value, new_value), sample_obj)
+        if key != "comment":
```

Comment:
maybe we should swith the order of the if-and else to avoid the negation

Reply:
Thanks! I removed the negation in the `if` statement.